src/ValueSchema/UnionSchema.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ScrumWorks\OpenApiSchema\ValueSchema;
+
+use ScrumWorks\OpenApiSchema\Exception\InvalidArgumentException;
+
+final class UnionSchema extends AbstractValueSchema
+{
+    /**
+     * @var ValueSchemaInterface[]
+     */
+    private array $possibleSchemas;
```

Comment:
je tam potreba to `possible`? jakoze chapu co tim chces rict, ale vicemene by mi prislo logicke mi klidne jen `schemas`... ale tohle neni nutne, spise je to otazka k zamysleni

Reply:
klidne muzeme prejmenovat, ale vyhledavat `schemas` je pak moc prijemne

Reply:
ok, tak asi nechat tak